keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -252,222 +252,465 @@ non-goals of this KEP.
 
 ## Proposal
 
-The basic idea is the following:
+On a high level, this proposal is based around two new concepts:
+
+1. Capacity pool, which is a construct for defining the capacities available for
+   sharing among partitions of a single physical device. Drivers/controllers will
+   typically create a capacity pool that specifies all the available capacities of
+   a physical device and then define the partitions available for DRA. Each
+   partition will specify how much capacity it will consume from the capacity
+   pool, which allows the scheduler to keep track of available capacity in
+   a pool and decide whether a partition can be allocated. A device will
+   separately define the capacities that are accessible to the user for the partition.
+
+1. Mixins, which allows devices and resource pools to reference a mixin
+   object that defines some properties and that when used, will extend the properties
+   they define explicitly on the device or resource pool. The objective is to reduce
+   the duplication and overall footprint of `ResourceSlices`.
+
+The implementation of these concepts requires several changes to the
+ResourceSlice API:
+
+1. It introduces four new fields on the `ResourceSliceSpec`. First, `CapacityPools`
+   defines a list of `CapacityPools`, each of which defines a set of capacities
+   that devices can draw from. `CapacityPoolMixins`, `DeviceMixins`, and
```

Comment:
Nit:

```suggestion
   defines a list of `CapacityPools`. `CapacityPoolMixins`, `DeviceMixins`, and
```

Details about `CapacityPools` are available below. Having one place to learn about them seems better to me.